### PR TITLE
refactor(grouped-character): move component from experimental - FE-2712

### PR DESCRIPTION
**Pull request by mkrds**
BREAKING CHANGE: GroupedCharacter has new import path -
carbon-react/lib/components/grouped-character
There is a codemod available to assist with this upgrade:
npx carbon-codemod move-experimental-components <target>
See https://github.com/Sage/carbon-codemod for more information.

### Proposed behaviour
This PR moves `GroupedCharacter` from `experimental` folder

### Checklist

- [x] Commits follow our style guide
- [x] Screenshots are included in the PR if useful
- [x] All themes are supported if required
- [x] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [x] Storybook added or updated if required
- [x] Typescript `d.ts` file added or updated if required
- [x] Carbon implementation and Design System documentation are congruent

### CODE REVIEW

**Review by sjohnson-sage — APPROVED** [review]

**Review by edleeks87 — APPROVED** [review]

**Review by edleeks87 — APPROVED** [review]

File: cypress/features/regression/accessibility/accessibilityCommon.feature [hunk 1]
```diff
@@ -32,6 +32,7 @@ Feature: Accessibility tests - Common list
       | Split-button        |
       | Step-sequence       |
       | Multi-action-button |
+      | groupedcharacter    |
```
Comment (DlgSHi) [inline cypress/features/regression/accessibility/accessibilityCommon.feature:35]: Could we put it alphabetically?

File: cypress/features/regression/accessibility/accessibilityCommon.feature [hunk 2]
```diff
@@ -52,4 +53,31 @@ Feature: Accessibility tests - Common list
       | DraggableContext |
       | Setting row      |
       | Table-ajax       |
-      | Table            |
\ No newline at end of file
+      | Table            |
+
```
Comment (DlgSHi) [inline cypress/features/regression/accessibility/accessibilityCommon.feature:57]: empty line

File: cypress/features/regression/experimental/groupedCharacter.feature [hunk 3]
```diff
@@ -1,9 +1,9 @@
-Feature: Experimental GroupedCharacter component
-  I want to check Experimental Grouped character component properties
+Feature: GroupedCharacter component
+  I want to check Grouped character component properties
 
   @positive
   Scenario Outline: Set groups to <nameOfObject> and verify input
-    Given I open default "Experimental GroupedCharacter Test" component in noIFrame with "groupedCharacter" json from "experimental" using "<nameOfObject>" object name
+    Given I open default "GroupedCharacter Test" component in noIFrame with "groupedCharacter" json from "experimental" using "<nameOfObject>" object name
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/groupedCharacter.feature:6]: We should also move the `cypress/fixtures/experimental/groupedCharacter.json` file into `cypress/fixtures/commonComponents` and change all import below

```suggestion
    Given I open default "GroupedCharacter Test" component in noIFrame with "groupedCharacter" json from "commonComponents" using "<nameOfObject>" object name
```

**Review by edleeks87 — APPROVED** [review]

**Review by sjohnson-sage — APPROVED** [review]

**Review by edleeks87 — APPROVED** [review]

**Review by sjohnson-sage — APPROVED** [review]

**Review by DlgSHi — APPROVED** [review]

**Pull request merged**